src/metabase/models/notification.clj

```diff
@@ -105,17 +106,16 @@
    notification-handlers
    k
    #(group-by :notification_handler_id
-              (let [recipients (t2/select :model/NotificationRecipient :notification_handler_id [:in (map :id notification-handlers)])
+              (let [recipients       (t2/select :model/NotificationRecipient
+                                                :notification_handler_id [:in (map :id notification-handlers)])
                     type->recipients (group-by :type recipients)]
                 (-> type->recipients
-                    (update :notification-recipient/user
-                            (fn [recipients]
-                              (when (seq recipients)
-                                (t2/hydrate recipients :user))))
-                    (update :notification-recipient/group
-                            (fn [recipients]
-                              (when (seq recipients)
-                                (t2/hydrate recipients [:permissions_group :members]))))
+                    (m/update-existing :notification-recipient/user
+                                       (fn [recipients]
+                                         (t2/hydrate recipients :user)))
```

Comment:
it'll also work like `(m/update-existing :notification-recipiet/user t2/hydrate :user)`, kind of nice :)